microservices/webhooks/index.js

```diff
@@ -0,0 +1,54 @@
+const express = require("express");
+const bodyParser = require("body-parser");
+const app = express();
+const superagent = require("superagent");
+const port = parseInt(process.env.PORT) || 8080;
+const token = process.env.WEBHOOKS_MICROSERVICE_TOKEN || "123456";
+const discordSponsorWebhook = process.env.DISCORD_SPONSOR_WEBHOOK || "";
```

Comment:
Same here, make it exit early with a error message/exit code if this can't be found, since its needed to run the thing